raft/testdata/confchange_v1.txt

```diff
@@ -0,0 +1,78 @@
+add-nodes 1 voters=(1) index=2
+----
+INFO 1 switched to configuration voters=(1)
+INFO 1 became follower at term 0
+INFO newRaft 1 [peers: [1], term: 0, commit: 2, applied: 2, lastindex: 2, lastterm: 1]
+
+campaign 1
+----
+INFO 1 is starting a new election at term 0
```

Comment:
We may want to clean up some of the log messages before we write too many tests that entrench them. For example, this "starting a new election" message is always paired with a "became candidate" message so there's no reason to have both.

Reply:
Thanks to `-rewrite`, it's not a problem when a log message changes. (My expectation is that none of the outputs are actually hand-written, at least mine never are). But I agree that the log messages need cleanup. Now that they are actually "exposed" in these tests I think we'll see a lot more of that, but I don't want to start going at it in this PR.